doc/source/whatsnew/v0.25.1.rst

```diff
@@ -54,7 +54,7 @@ Numeric
 ^^^^^^^
 - Bug in :meth:`Series.interpolate` when using a timezone aware :class:`DatetimeIndex` (:issue:`27548`)
 - Bug when printing negative floating point complex numbers would raise an ``IndexError`` (:issue:`27484`)
--
+- Bug where operators on DataFrames and Series with axis=1 would fail on large dataframes(:issue:`27636`)
```

Comment:
Is there a way to be more descriptive here? Haven't read through issue but on initial glance it's not clear what fails

Reply:
Changed to be more specific to the issue.